components/automate-ui/src/app/modules/infra-proxy/data-bags-details/data-bags-details.component.scss

```diff
@@ -73,9 +73,28 @@ chef-loading-spinner {
 
 chef-tab-selector {
   chef-option {
-    margin-right: 16px;
+    margin-right: 4px;
     color: $chef-primary-dark;
   }
+
+  ::ng-deep .selected {
+    background-color: #F3F6F8;
+  }
+
+  ::ng-deep .selected::before {
+    height: 4px;
+    background-color: #3864F2;
+  }
+
+  chef-option:focus {
+    outline: none;
+  }
+
+  ::ng-deep chef-option::before {
+    height: 4px;
+    background-color: #E7E9E9;
+  }
```

Comment:
I see this repeated over and over, which is a code smell. Consider extracting a new sub-component to encapsulate this behavior and use it here in each of these components.